docker-images/kafka-based/kafka/scripts/kafka_run.sh

```diff
@@ -110,7 +110,13 @@ fi
 
 KEY_STORE=/tmp/kafka/cluster.keystore.p12
 TRUST_STORE=/tmp/kafka/cluster.truststore.p12
-KAFKA_OPTS="${KAFKA_OPTS} -javaagent:$(ls "$KAFKA_HOME"/libs/kafka-agent*.jar)=$KAFKA_READY:$ZK_CONNECTED:$KEY_STORE:$CERTS_STORE_PASSWORD:$TRUST_STORE:$CERTS_STORE_PASSWORD"
+
+echo "sslKeyStorePath=$KEY_STORE" > /tmp/kafka-agent.properties
+echo "sslKeyStorePass=$CERTS_STORE_PASSWORD" >> /tmp/kafka-agent.properties
+echo "sslTrustStorePath=$TRUST_STORE" >> /tmp/kafka-agent.properties
+echo "sslTrustStorePass=$CERTS_STORE_PASSWORD" >> /tmp/kafka-agent.properties
```

Comment:
This part of the code failed the build because of Shellcheck. Maybe you can do something like this to make it more easily readable and make it pass shellcheck?

```suggestion
tee /tmp/kafka-agent.properties <<EOF
sslKeyStorePath=${KEY_STORE}
sslKeyStorePass=${CERTS_STORE_PASSWORD}
sslTrustStorePath=${TRUST_STORE}
sslTrustStorePass=${CERTS_STORE_PASSWORD}
EOF
```

Should we try to delete the file first to make sure it does not exist?